posts/just-a-developer.md

```diff
@@ -0,0 +1,29 @@
+---
+{
+  "title": "Just a Developer",
+  "author": "Terry Keeney",
+  "tags": ["thoughts","rant"],
+  "published": "Fri Jan 13 2017 18:03:43 GMT-0700 (MST)"
+}
+---
+
+
+# Just a Developer
+
+The explosion of languages, frameworks and platforms in the software development world has created a bit of a double edged sword for those fancying themselves as technical generalists capable of doing just about anything.  On the one hand it's created a world of ever more shiny new things offering a limitless supply of new toys to experiment with.  On the other it's highlighted the stark difference between what I'll classify as coders and developers - making interactions with recruiters and other technical-interfacing leypersons increasingly more infuriating.
+
+In my grossly generalized world the Coder can be identified by their battling cry: "That's a &lt;not-my-tech&gt; problem, I'm a &lt;my-tech-proficiency&gt; developer".  The Coder is characterized by his proficiency in a narrow range of chosen skills.  The Coder will progresses and becomes better in his niche by sheer practice and repetition alone.  
+
+By contrast the Developer's single greatest skill is in being an applied learner.  Quite simply the Developer has a knack for grokking new [languages|frameworks|platoforms] and becoming proficient very quickly.  There is no problem beyond there reach for they enjoy building at any level stack.
```

Comment:
their instead of "there" last sentence.  And maybe that sentence needs a comma after reach?